docs/attributes-registry/process.md

```diff
@@ -31,7 +32,26 @@
 | `process.session_leader.pid` | int | The PID of the process's session leader. This is also the session ID (SID) of the process. | `14` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `process.user.id` | int | The effective user ID (EUID) of the process. | `1001` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `process.user.name` | string | The username of the effective user of the process. | `root` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `process.vpid` | int | Virtual process identifier. [1] | `12` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `process.vpid` | int | Virtual process identifier. [2] | `12` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
-**[1]:** The process ID within a PID namespace. This is not necessarily unique across all processes on the host but it is unique within the process namespace that the process exists within.
+**[1]:** A globally unique identifier of the process. This can be any format
+of unique indentifier.
+
+On Linux, where possible, it is RECOMMENDED that this is calculated
+MD5 hash of the string `<pid>_<starttime>_<bootid>`.
+
+e.g.
+
+| Key | Description | Example Value |
+| --- | --- | --- |
+| `pid` | PID | "1201" |
+| `starttime` | start time in seconds since boot | "3029" |
```

Comment:
```suggestion
| `starttime` | The number of seconds since the last boot | "3029" |
```

Reply:
I've change this to `The process start time in seconds since system boot`, just to make it clear that's what time it's referring to